src/lib/models/invoice.js

```diff
@@ -187,6 +196,12 @@ class Invoice extends Model {
   get isDeMinimis () {
     return this._isDeMinimis;
   }
+
+  toJSON () {
```

Comment:
This `toJSON` method doesn't appear to add anything to the functionality of the super class, so could probably be safely removed

Reply:
Removed